examples/lwt_unix_doc/docker_lwt.ml

```diff
@@ -1,14 +1,6 @@
 open Lwt.Infix
 open Cohttp_lwt_unix
 
-let ctx =
```

Comment:
We should leave this here and instead replace `t` with
```ocaml
let t =
  Cohttp_lwt_unix.Client.get ~ctx (Uri.of_string "http://docker/version")
  >>= fun (resp, body) ->
  let open Cohttp in
  let code = resp |> Response.status |> Code.code_of_status in
  Printf.printf "Response code: %d\n" code;
  Printf.printf "Headers: %s\n" (resp |> Response.headers |> Header.to_string);
  body |> Cohttp_lwt.Body.to_string >|= fun body ->
  Printf.printf "Body of length: %d\n" (String.length body);
  print_endline ("Received body\n" ^ body)
```
This other body was probably an oversight.
Or at least add a flag to select one of the other behaviour.

These examples were meant to build the examples in the README.
There is a ticket to move to `mdx`, but until that is implemented, we should try and keep them for now.

Reply:
Done.